src/telemetryGenerator.node.ts

```diff
@@ -0,0 +1,784 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import * as ts from 'typescript';
+import * as fs from 'fs-extra';
+import glob from 'glob';
+import { initialize } from './test/vscode-mock';
+import { Parser } from 'json2csv';
+
+initialize();
```

Comment:
Lots of changes and improvements:
* We now generate the telemetry.md file with everything, including the properties, comments and the like
* We generate a CSV so we can query/filter the telemetry
* We generate the necessary stuff for GDPR tooling